ACP2E-2187: [Cloud] Youtube video autoplay is not working (#130)
AdobeDocs/commerce-frontend-core: src/pages/guide/themes/product-video.md

1. When complete, click _Save Config_.

1. When prompted, refresh the cache.

<InlineAlert variant="info" slots="text"/>

> If the **Autostart base video** configuration option is set to `Yes` but the video does not begin to play automatically, it could be due to the autoplay policies that are enforced by the browser and cannot be controlled by Adobe Commerce. Each supported browser has its own autoplay policies that can change over time and your video may not autoplay in the future. As a recommended best practice, you should not rely on autoplay for business critical functionality and should test the video autoplay behavior in your store with each supported browser.

Reviewer: ```suggestion
If the **Autostart base video** configuration option is set to `Yes` but the video does not begin to play automatically, it could be due to the autoplay policies that are enforced by the browser and cannot be controlled by Adobe Commerce. Each supported browser has its own autoplay policies that can change over time and your video may not autoplay in the future. As a recommended best practice, you should not rely on autoplay for business critical functionality and should test the video autoplay behavior in your store with each supported browser.

```

1. When complete, click _Save Config_.

1. When prompted, refresh the cache.

<InlineAlert variant="info" slots="text"/>

> If you have "Autostart base video" configuration option set to Yes, but the video does not begin to play automatically, it could be due to autoplay policies that are enforced by the browser and cannot be controlled by Adobe Commerce.

Reviewer: ```suggestion
If the **Autostart base video** configuration option is set to `Yes` but the video does not begin to play automatically, it could be due to the autoplay policies that are enforced by the browser and cannot be controlled by Adobe Commerce. Each supported browser has its own autoplay policies that can change over time and your video may not autoplay in the future. As a recommended best practice, you should not rely on autoplay for business critical functionality and should test the video autoplay behavior in your store with each supported browser.
```

> 

Reviewer: ```suggestion
```

> Note that each supported browser has its own autoplay policies that changes over time and your video may not autoplay in the future. We recommend that you do not rely on autoplay for business critical functionality and test the video autoplay behavior in your store with each supported browser.

Reviewer: ```suggestion
```
